Source/WebKit/UIProcess/WebAuthentication/Authenticator.h

```diff
@@ -50,6 +50,9 @@ using AuthenticatorObserverRespond = std::variant<Ref<WebCore::AuthenticatorResp
 class AuthenticatorObserver : public CanMakeWeakPtr<AuthenticatorObserver> {
 public:
     virtual ~AuthenticatorObserver() = default;
+
+    DECLARE_VIRTUAL_REFCOUNTED;
```

Comment:
The code was correct before. No reason to hide behind a macro.

Reply:
If macro exists, then we should use the macro for consistency.

Reply:
I believe the plan is to move away from the macro and remove it.

Reply:
This macro was introduced by Geoff just three weeks ago. https://github.com/WebKit/WebKit/commit/5d100c24f90237a63172d016c390a3bd223293e6

Reply:
I know and I've objected to this macro since its introduction. It was discussed during last Thursday Safer C++ Weekly sync.

Reply:
Oh, I see. Okay, I. revert this for now.